contracts/modules/SwapsExternal.sol

```diff
@@ -30,15 +31,19 @@ contract SwapsExternal is VaultController, SwapsUser {
         address returnToSender,
         uint256 sourceTokenAmount,
         uint256 requiredDestTokenAmount,
-        bytes calldata swapData
+        uint256 minReturn,
+        bytes calldata swapData,
+        address priceFeeds_
```

Comment:
why the caller controls `priceFeeds_`? it looks dangerous. Feels like oracle manipulation for free.